File: lab02/zad03.m
R = 0.01;
u = load('u.dat');
P = mean(u .^ 2) / R;

l10 = length(u) / 10;
Pc = zeros(1, length(u)-l10);

for i=1:length(Pc)
    Pc(i) = mean(u(i:i+l10-1) .^ 2) / R;
end

File: lab02/zad03.py
import numpy as np

R = 0.01
u = np.loadtxt('u.dat')
P = np.mean(np.abs(u)**2) / R
print(P)

l10 = int(len(u) / 10)
Pc = np.zeros(len(u) - l10 + 1)

for i in range(0, len(Pc)):
    Pc[i] = np.mean(np.abs(u[i:i+l10]**2)) / R

print(np.max(Pc))
